.circleci/config.yml

```diff
@@ -166,11 +166,13 @@ commands:
         type: string
     steps:
       - attach_workspace:
-          at: ~/mattermost-mobile
-      - deploy:
+          at: .
+      - attach_workspace:
+          at: ~/mattermost-mobile-private
```

Comment:
does it make sense to have the workspace attached in two places?
not sure if it would be better to use a symlink 0/5 on this.

Reply:
how would you use a symlink here?